docs/tutorials/streaming-replication.mdx

```diff
@@ -0,0 +1,162 @@
+---
+title: Set up streaming replication
+description: Use Xata's streaming replication to keep your database continuously synchronized with real-time changes.
+---
+
+This guide shows you how to set up continuous streaming replication from your production PostgreSQL database to Xata, enabling real-time data synchronization with optional anonymization.
+
+![Setting up streaming replication to Xata](assets/images/xata-streaming-replication.png)
+
+## 1. Prerequisites
+
+- A Xata account ([sign up here](https://console.xata.io))
+- The [Xata CLI](/cli) installed:
+  ```bash
+  curl -fsSL https://xata.io/install.sh | bash
+  ```
+- A PostgreSQL database with:
+  - Logical replication enabled
+  - A dedicated replication slot (or permissions to create one)
+  - Network connectivity from Xata to your database
+
+## 2. Enable logical replication on source database
+
+First, ensure your source PostgreSQL database has logical replication enabled. You'll need to set these parameters:
+
+```sql
+-- Check current settings
+SHOW wal_level;
+SHOW max_replication_slots;
+SHOW max_wal_senders;
+```
+
+If not already configured, update your PostgreSQL configuration:
+
+```sql
+ALTER SYSTEM SET wal_level = logical;
+ALTER SYSTEM SET max_replication_slots = 10;
+ALTER SYSTEM SET max_wal_senders = 10;
+```
+
+Restart your PostgreSQL instance for the changes to take effect.
+
+## 3. Create a Xata project and branch
+
+In the Console, create a new project and then click the **Create main branch** button to create the PostgreSQL instance.
+
+For streaming replication, consider using at least 1 replica to ensure high availability during continuous synchronization. Select an instance size that can handle your expected write throughput.
+
+> **Note:** Streaming replication maintains a persistent connection to your source database. Ensure your network allows stable, long-lived connections between Xata and your PostgreSQL instance.
+
+## 4. Configure the Xata CLI
+
+Authenticate the CLI by running:
+
+```sh
+xata auth login
+```
+
+Initialize the project by running:
+
+```sh
+xata init
+```
+
+## 5. Configure streaming replication
+
+Generate a configuration for the streaming process:
+
+```bash
+xata clone config --source-url $CONN_STRING
+```
+
+Where `CONN_STRING` is your PostgreSQL connection string with replication permissions.
+
+The configuration prompt will ask you to:
+
+- Select tables to replicate
+- Configure anonymization rules if needed
```

Comment:
Wouldn't the anonymization rules and the transformation pipelines be the same thing?